bbschema/entity.py

```diff
@@ -343,9 +343,10 @@ def update(self, identifier_json):
 
         if 'value' in identifier_json:
             new.value = identifier_json['value']
-        if 'identifier_type_id' in identifier_json:
+
+        if 'identifier_type' in identifier_json:
             new.identifier_type_id =\
-                identifier_json.get('identifier_type', {}).\
+                identifier_json.get('identifier_type').\
```

Comment:
As you pointed out, we check for this before trying to access it, so we can just do:
    identifier_json['identifier_type'].get('identifier_type_id')